backend/src/scraper/bestbuy_scraper.py

```diff
@@ -109,9 +110,11 @@ def _product_reviews(product_id: str, product_name: str) -> Dict:
                         review_results = search_sku.json()
                         reviews.extend(review_results.get("reviews", []))
 
-                # TODO: Get most positive and most negative reviews.
-                top_positive = {}
-                top_negative = {}
+                promo_string = "[This review was collected as part of a promotion.] "
+                for r in reviews:
+                    r["comment"] = r.get("comment").replace(promo_string, "")
```

Comment:
I haven't seen any without the space. I think it's pretty certain since it seems to be tacked onto the review automatically. I think they would all follow the same format.